docs/about.rst

```diff
@@ -0,0 +1,5 @@
+*******************
+Project Description
+*******************
+
+.. mdinclude:: ../README.md
```

Comment:
this is how you can embed markdown files within `rst` files and "link back" to documentation in the root directory.